components/profile/components/EditProfile.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const EditProfile = () => {
+	return <div>Profile Edit</div>;
+};
```

Comment:
you should be getting an error for doing this exact pattern with an arrow function. If you have no other code than the the return you the implicit arrow return pattern
```
const EditProfile = () => <div>Profile Edit</div>;
```